unstructured/embed/huggingface.py

```diff
@@ -66,15 +65,3 @@ def _add_embeddings_to_elements(self, elements, embeddings) -> List[Element]:
             element.embeddings = embeddings[i]
             elements_w_embedding.append(element)
         return elements
-
-    @EmbeddingEncoderConnectionError.wrap
```

Comment:
Is this okay to totally remove?

Reply:
This logic should be moved to the ingest repo anyways.

Reply:
Got it, thanks!